pep/app/styles/pages/browse.scss

```diff
@@ -0,0 +1,21 @@
+.browse {
+    .volume {
+        min-width: 125px;
+        max-width: 125px;
+    }
+    .dotted-line {
+        position: absolute;
+        border-bottom: dotted 1px $black;
+        width: 100%;
+        height: 100%;
+    }
+
+    .page {
+        width: 35px;
```

Comment:
should probably use `rem` for dimensions like this when we can (i.e. this is probably about `2rem`). when `rem` is used universally for styles it makes scaling the UI to different base font sizes (which we'll eventually need for the user font size preference) much easier.